config/lava/chromeos/chromeos-tast-template.jinja2

```diff
@@ -0,0 +1,45 @@
+{% extends 'base/kernel-ci-base.jinja2' %}
+{% block actions %}
+{% set extra_kernel_args = "console_msg_format=syslog earlycon" cros_debug cros_secure root=/dev/mmcblk0p3 %}
```

Comment:
There's a problem with the double-quotes there.

Reply:
finally it works again..
New lava job to test: https://lava.collabora.co.uk/scheduler/job/5794516

Reply:
Well, the job above landed on octopus cbg-2 which doesn't have a Chrome OS image install. So that's why it failed. If you resubmit it with a tag set to cbg-0 or cbg-1 it should work.

Also someone should flash all the octopus devices with a Chrome OS image to avoid this problem, ideally with a job kept under source control. @mgalka Any news on this front?

Reply:
cbg-0 : https://lava.collabora.co.uk/scheduler/job/5801891
cbg-1: https://lava.collabora.co.uk/scheduler/job/5801894

Reply:
Great. There's a problem with the Chrome OS setup on these devices, but that's a known issue:
```
Process exited with status 127 (output: \"bash: line 0: exec: file: not found\n\")
```
That's because /usr/local is not mounted. It's not an issue with this PR, so I think it's working as expected.